sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Custom/BackupProtectedItemCollection.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.RecoveryServicesBackup
+{
+    /// <summary> The BackupProtectedItemCollection. </summary>
+    public partial class BackupProtectedItemCollection
+    {
+        /// <summary>
+        /// <summary>Create or Update.</summary>
+        /// </summary>
+        /// <param name="waitUntil"></param>
+        /// <param name="protectedItemName"></param>
+        /// <param name="data"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
```

Comment:
Could we copy back the doc for this method from our git history?

Reply:
Please do the same for all other methods we added in customization code.

Reply:
Fixed in latest commit.